### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by 0pcom**
* how to compile and install from source code complete with steps

* how to install golang and other necessary tools

* e̶x̶p̶l̶a̶i̶n̶ ̶h̶o̶w̶ ̶t̶o̶ ̶i̶n̶s̶t̶a̶l̶l̶ ̶t̶h̶e̶ ̶i̶n̶s̶t̶a̶l̶l̶e̶r̶ ̶v̶e̶r̶s̶i̶o̶n̶s̶.̶ ̶I̶ ̶t̶h̶i̶n̶k̶ ̶o̶n̶ ̶W̶i̶n̶d̶o̶w̶s̶ ̶a̶n̶d̶ ̶M̶a̶c̶ ̶i̶t̶s̶ ̶j̶u̶s̶t̶ ̶d̶r̶a̶g̶ ̶a̶n̶d̶ ̶d̶r̶o̶p̶ ̶s̶o̶ ̶i̶t̶s̶ ̶n̶o̶t̶ ̶m̶u̶c̶h̶ ̶t̶o̶ ̶e̶x̶p̶l̶a̶i̶n̶ ̶b̶u̶t̶ ̶D̶e̶b̶i̶a̶n̶ ̶n̶e̶e̶d̶s̶ ̶t̶o̶ ̶b̶e̶ ̶e̶x̶p̶l̶a̶i̶n̶e̶d̶ ̶w̶i̶t̶h̶ ̶a̶c̶c̶u̶r̶a̶t̶e̶ ̶c̶o̶m̶m̶a̶n̶d̶s̶
The package documentation is in the wiki, in case it needs to be updated on short notice.
Intentionally not included in the readme, but a link to the package installation guide is included
The installers for windows and mac should be documented by @mrpalide or someone who uses those operating systems

* removed most of the package build info

* c̶r̶e̶a̶t̶i̶n̶g̶ ̶a̶ ̶G̶i̶t̶h̶u̶b̶ ̶r̶e̶l̶e̶a̶s̶e̶ ̶c̶a̶n̶ ̶a̶l̶s̶o̶ ̶b̶e̶ ̶m̶o̶v̶e̶d̶ ̶t̶o̶ ̶t̶h̶e̶ ̶W̶i̶k̶i̶ ̶a̶l̶t̶h̶o̶u̶g̶h̶ ̶I̶ ̶k̶n̶o̶w̶ ̶w̶e̶ ̶h̶a̶d̶ ̶i̶t̶ ̶i̶n̶ ̶t̶h̶e̶ ̶R̶E̶A̶D̶M̶E̶ ̶f̶o̶r̶e̶v̶e̶r̶
I think we should keep this in the README, it's just a little section at the end. If you insist, it can be removed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,18 +108,22 @@ make format check
...
-To compile skywire directly from this repository for local testing and development
+To compile skywire directly from cloned git sources
 
 ```
 git clone https://github.com/skycoin/skywire
 cd skywire
 #for the latest commits, check out the develop branch
 git checkout develop
-make build
+make build1
```
Comment (jdknives) [inline README.md:120]: Typo I would assume.

Comment (jdknives) [reply README.md:120]: I hope we did not introduce multiple build targets like `build1` `build2`...

**Review by jdknives — COMMENTED** [review]
The lack of proper capitalization is bugging me a bit, but in terms of the changes requested this looks mostly fine. I dont understand why we have not moved the Package building instructions to the wiki.

File: README.md (hunk 1 above)
Comment (0pcom) [reply README.md:120]: > I hope we did not introduce multiple build targets like `build1` `build2`...

synth objected to the binaries populating in the repository root, he wanted these in `./build`

so `make build1` preserves the previous output / behavior of `make build`

`make build` now creates the binaries as specified inside `./build`

**Comment by 0pcom** [comment]
> The lack of proper capitalization is bugging me a bit, but in terms of the changes requested this looks mostly fine. I dont understand why we have not moved the Package building instructions to the wiki.

I got rid of most of the references to this and just left the section about linux packages in case someone needed to know how to do that.

The information is important enough that I think it deserves to stay.

It describes installing the build variants on archlinux as well as how to create the debian packages.

Any maintainer must know how to create the debian packages.

If something happens to me, this information need not be hidden in the wiki. It's at the same level as creating a github release and it's necessary to do that immediately after any new release is created.

**Comment by 0pcom** [comment]
I've capitalized and added punctuation with the last commit

**Pull request merged**